include/text.php

```diff
@@ -1169,36 +1169,45 @@ function redir_private_images($a, &$item)
 	}
 }
 
+/**
+ * Sets the "rendered-html" field of the provided item
+ *
+ * Body is preserved to avoid side-effects as we modify it just-in-time for spoilers and private image links
+ *
+ * @param array $item
+ * @param bool  $update
+ *
+ * @todo Remove reference, simply return "rendered-html"
```

Comment:
We need both "rendered-html" and "rendered-hash" in return.

Reply:
Obviously this functionality needs some severe reworking :-) It was implanted some time when I hadn't got the knowledge about the system like I'm having it now.